### PR TITLE
adds PrivacyAgreement component

### PR DESCRIPTION
**Pull request by lbai001**
adds PrivacyAgreement component
updates ErrorableCheckbox component

### CODE REVIEW

File: src/components/form/controls/ErrorableCheckbox/ErrorableCheckbox.jsx [hunk 1]
```diff
@@ -80,33 +96,12 @@ class ErrorableCheckbox extends React.Component {
 }
 
 ErrorableCheckbox.propTypes = {
-  /**
```
Comment (jbalboni) [inline src/components/form/controls/ErrorableCheckbox/ErrorableCheckbox.jsx]: I think we want to keep these comments.

File: src/components/form/controls/ErrorableCheckbox/ErrorableCheckbox.unit.spec.jsx [hunk 2]
```diff
@@ -103,5 +140,5 @@ describe('<ErrorableCheckbox/>', () => {
     expect(inputs.prop('id')).to.equal(labels.prop('htmlFor'));
   });
 
-/* eslint-enable no-unused-vars */
+  /* eslint-enable no-unused-vars */
```
Comment (jbalboni) [inline src/components/form/controls/ErrorableCheckbox/ErrorableCheckbox.unit.spec.jsx]: Can we remove this line altogether?

File: src/components/PrivacyAgreement/PrivacyAgreement.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { mount } from 'enzyme';
+import { axeCheck } from '../../../lib/testing/helpers';
+import PrivacyAgreement from './PrivacyAgreement.jsx';
+
+chai.use(chaiAsPromised);
+const { expect } = chai;
+/* eslint-disable no-unused-vars */
```
Comment (jbalboni) [inline src/components/PrivacyAgreement/PrivacyAgreement.unit.spec.jsx]: What is triggering the linting error that we're ignoring here? Could we remove this line?

File: src/components/PrivacyAgreement/PrivacyAgreement.unit.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { mount } from 'enzyme';
+import { axeCheck } from '../../../lib/testing/helpers';
+import PrivacyAgreement from './PrivacyAgreement.jsx';
+
+chai.use(chaiAsPromised);
```
Comment (jbalboni) [inline src/components/PrivacyAgreement/PrivacyAgreement.unit.spec.jsx]: I don't think this is being used.

File: src/components/PrivacyAgreement/PrivacyAgreement.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            <a target="_blank" href="/privacy/">
+              privacy policy
+            </a>
+          </span>
+        }/>
+    </div>
+  );
+}
+
+PrivacyAgreement.propTypes = {
```
Comment (jbalboni) [inline src/components/PrivacyAgreement/PrivacyAgreement.jsx]: Should add some comments in here for these props.

File: src/components/PrivacyAgreement/PrivacyAgreement.unit.spec.jsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        onChange={e => {
+          e.target.checked;
+        }}/>
+    );
+  });
+  it('should render', () => {
+    expect(wrapper.text()).to.eql('I have read and accept the privacy policy*');
+  });
+
+  it('should pass aXe check', () => {
```
Comment (jbalboni) [inline src/components/PrivacyAgreement/PrivacyAgreement.unit.spec.jsx]: Can we also run the axe check when there's an error message being shown?

Comment (lbai001) [reply src/components/PrivacyAgreement/PrivacyAgreement.unit.spec.jsx]: okay

File: src/components/PrivacyAgreement/PrivacyAgreement.jsx (hunk 5 above)
Comment (lbai001) [reply src/components/PrivacyAgreement/PrivacyAgreement.jsx]: okay

File: src/components/PrivacyAgreement/PrivacyAgreement.unit.spec.jsx (hunk 3 above)
Comment (lbai001) [reply src/components/PrivacyAgreement/PrivacyAgreement.unit.spec.jsx]: this is from another file, i'll remove this

File: src/components/form/controls/ErrorableCheckbox/ErrorableCheckbox.unit.spec.jsx (hunk 2 above)
Comment (lbai001) [reply src/components/form/controls/ErrorableCheckbox/ErrorableCheckbox.unit.spec.jsx]: yes

File: src/components/form/controls/ErrorableCheckbox/ErrorableCheckbox.jsx (hunk 1 above)
Comment (lbai001) [reply src/components/form/controls/ErrorableCheckbox/ErrorableCheckbox.jsx]: will do

**Pull request merged**